### PR TITLE
Add newsletter November

### PR DESCRIPTION
**Pull request by wipfli**
Newsletter November. Cool stuff this month...

### CODE REVIEW

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Multi-sprites support for MapLibre Native](https://github.com/maplibre/maplibre-native/pull/1858) is expected to land soon. More information about this feature can be found in the [MapLibre Style Spec documentation](https://maplibre.org/maplibre-style-spec/sprite/).
+
+- Two design proposals were accepted this month:
+  - [Complex, Animatable, Interactive Annotations Proposal](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2023-11-08-complex-animatable-interactive-annotations.md)
+  - [Ambient cache initialization speed improvement](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2023-11-06-ambient-cache-initialization-speed-improvement.md)
+
+## Martin
+
+The new Martin tile server now has a tool similar to `tilelive-copy` to bulk copy large amounts of tiles from any source supported by the Martin tile server, using the same source configuration system/CLI parameters, but without starting up the whole server in order to generate them.
```
Comment (nyurik) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:56]:
```suggestion
Martin tile server now has a tool similar to `tilelive-copy` and `tippecanoe` to bulk copy tiles in bulk from any source supported by the Martin tile server, using the same source configuration system/CLI parameters, but without starting up the whole tile server in order to generate them.
```

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Use case: generate several areas of the map from PostgreSQL tables/functions or other sources, and even merge multiple tile layers together into a single tileset and store them into an .mbtiles file.
```
Comment (nyurik) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:58]:
```suggestion
Use case: generate several areas of the map from PostgreSQL tables/functions or other sources, and even merge multiple tile layers together into a single .mbtiles tileset file.
```

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Another new tool improvement: `mbtiles summary` prints now in-depth details about an mbtiles file.
```
Comment (nyurik) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:60]:
```suggestion
In other news, `mbtiles summary` tool can now print in-depth details about an mbtiles file such as per-zoom statistics.
```

**Review by nyurik — COMMENTED** [review]
a few minor improvements to Martin text

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+### Globe
+
+Not just one, but actually two proofs of concept for a globe view were shared by two independent developers this month. See https://github.com/maplibre/maplibre/discussions/161.
+
+We are excited to see where this will bring us!
+
+<a href="https://pheonor.github.io/globe-demo/"><img src="globe.jpg" style="max-width: 100%"></a>
+<br>
+
+<i>Globe view proof-of-concept by Vivan Larrieu. Link: https://pheonor.github.io/globe-demo/.</i>
```
Comment (louwers) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:28]:
```suggestion
<i>Globe view proof of concept by Vivan Larrieu. Link: https://pheonor.github.io/globe-demo/.</i>
```

**Review by louwers — APPROVED** [review]

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+---
+title: "MapLibre Newsletter November 2023"
+date: "2023-11-30"
+categories: ["announcements"]
+authors: [wipfli]
```
Comment (louwers) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:5]:
```suggestion
authors: [wipfli, yuri, bart]
```

File: content/news/2023-11-30-maplibre-newsletter-november-2023/index.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Multi-sprites support for MapLibre Native](https://github.com/maplibre/maplibre-native/pull/1858) is expected to land soon. More information about this feature can be found in the [MapLibre Style Spec documentation](https://maplibre.org/maplibre-style-spec/sprite/).
+
+- Two design proposals were accepted this month:
+  - [Complex, Animatable, Interactive Annotations Proposal](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2023-11-08-complex-animatable-interactive-annotations.md)
+  - [Ambient cache initialization speed improvement](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2023-11-06-ambient-cache-initialization-speed-improvement.md)
+
+## Martin
+
+Martin tile server now has a tool similar to `tilelive-copy` and `tippecanoe` to copy tiles in bulk from any source supported by the Martin tile server, using the same source configuration system/CLI parameters, but without starting up the whole tile server in order to generate them. Available as a standalone binary with no external dependencies.
```
Comment (nyurik) [inline content/news/2023-11-30-maplibre-newsletter-november-2023/index.md:56]:
```suggestion
Martin tile server has a new tool similar to `tilelive-copy` and `tippecanoe` to bulk-copy tiles from any supported source. The `martin-cp` tool uses the same source configuration system/CLI parameters as Martin server, but without starting up the whole tile server in order to generate them. The tool is available as a standalone binary with no external dependencies.
```

**Pull request merged**